src/control/control/nodes/test_path.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+from nav_msgs.msg import Odometry
+import rclpy
+from rclpy.node import Node
+
+class TestPath(Node):
+
+    def __init__(self):
+        super().__init__('test_path')
+        self.path_publisher = self.create_publisher(Odometry, 'path', 10)
+        self.timer = self.create_timer(1.0, self.publish_path)  # Timer to publish every 1 second
+        self.x_position = 10.0  # Initial x position
+
+    def publish_path(self):
+        msg = Odometry()
```

Comment:
Why is this odometry and not generated path?

Reply:
This seems like a PID to a waypoint and not a path

Reply:
I believe we discussed this at the meeting. If not, I believe that it is best to integrate path following as a separate PR.